iree/compiler/Conversion/Common/LinalgBufferizePass.cpp

```diff
@@ -688,10 +689,27 @@ static Value getInplaceResultBuffer(OpBuilder &b, OpResult resultValue,
     user = *(defVal.user_begin());
     // If the user is a store op, we are done.
     if (isa<IREE::Flow::DispatchTensorStoreOp>(user)) break;
-    // If user has more than one results, abort.
-    if (user->getNumResults() != 1) return nullptr;
+    // If user has more than one results, with a `LinalgOp` we can still follow
+    // the chain. For now use the `tiedOperands` cause that might result in
+    // better reuse tracking.
+    if (auto linalgOp = dyn_cast<linalg::LinalgOp>(user)) {
+      OpOperand &use = *defVal.use_begin();
+      auto tiedOperands = getTiedOperandsForLinalgOps(linalgOp);
+      Value useResult = nullptr;
```

Comment:
[optional] we don't need to assign `nullptr`. It is `nullptr` without any assignment.

Reply:
Yeah, but it just makes it easier to read.